LayoutTests/fast/attachment/mac/attachment-keynote-icon-expected.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html><!-- webkit-test-runner [ AttachmentElementEnabled=true ] -->
+<html class="reftest-wait">
+<body>
+<attachment></attachment>
+<script>
+if (window.internals)
+    file = window.internals.createFile("../resources/test-file.key");
+
+var attachments = document.getElementsByTagName("attachment");
+for (var i = 0; i < attachments.length; i++)
+    attachments[i].file = file;
```

Comment:
we probably don't need to loop over a single item here?

`document.querySelector("attachment").file = file;`

Reply:
That's from the original file. Maybe it used to test multiple attachments? Anyway, I guess I can simplify all of them since I'm heavily modifying the expected files and creating copies.

Reply:
Spoke too fast, it's actually from other similar tests I pilfered, and some of them do have multiple attachments. I'll focus on iWork tests in this patch.